rust/connlib/clients/android/src/lib.rs

```diff
@@ -71,8 +70,8 @@ pub unsafe extern "system" fn Java_dev_firezone_connlib_Session_connect(
         "(Ljava/lang/String;)Z",
         &[JValue::from(&tunnel_addresses)],
     ) {
-        Ok(res) => trace!("onSetTunnelAddresses returned {:?}", res),
-        Err(e) => error!("Failed to call setTunnelAddresses: {:?}", e),
+        Ok(res) => log::trace!("`onSetTunnelAddresses` returned `{res:?}`"),
+        Err(err) => log::error!("Failed to call `setTunnelAddresses`: {err}"),
```

Comment:
+1 for referencing `trace!` etc with `log::`. I much prefer that too because it avoids import lines and actually reads quite nice.